src/components/SideBar/SideBar.tsx

```diff
@@ -10,14 +10,18 @@ type ApiProp = {
 }
 
 export const SideBar = ({apiList, toggleFavorite}:ApiProp) => {
+  const toggleSideBar = () => {
+    document.querySelector('.side-bar')?.classList.toggle('open');
+    document.querySelector('.arrow')?.classList.toggle('open-arrow')
+  }
 
   return (
     <div className='side-bar'>
-      <GiHamburgerMenu 
-      className="hamburger"
-      onClick={() => document.querySelector('.side-bar')?.classList.toggle('open')}></GiHamburgerMenu>
+      <HiArrowRight 
+      className="arrow"
+      onClick={toggleSideBar} />
```

Comment:
Good call breaking out this event listener to its own function to get rid of return functionality